src/app/containers/ConsentBanner/Banner.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+import { ServiceContextConsumer } from '../../contexts/ServiceContext';
+import {
+  ConsentBanner,
+  ConsentBannerButton,
+  ConsentBannerLink,
+  ConsentBannerText,
+} from '../../components/ConsentBanner';
+
+const Accept = (message, onClick) => (
+  <ConsentBannerButton onClick={onClick}>{message}</ConsentBannerButton>
+);
+
+const Reject = (message, href, onClick) => (
+  <ConsentBannerLink href={href} onClick={onClick}>
+    {message}
+  </ConsentBannerLink>
+);
+
+const Text = ({ first, linkText, linkUrl, last }) => {
+  const Link = linkUrl && linkText ? <a href={linkUrl}>{linkText}</a> : null;
+
+  return (
+    <ConsentBannerText>
+      {first}
+      {Link}
+      {last}
+    </ConsentBannerText>
+  );
+};
+
+Text.propTypes = {
+  first: string.isRequired,
+  linkText: string.isRequired,
+  linkUrl: string.isRequired,
+  last: string.isRequired,
+};
+
+const ConsentBannerContainer = ({ type, onReject, onAccept }) => (
+  <ServiceContextConsumer>
+    {({ translations }) => {
+      const messaging = translations.consentBanner[type];
```

Comment:
```suggestion
      const consentBannerConfig = translations.consentBanner[type];
```

This then means the following lines will be easier to read going from: 
```
title={messaging.title}
reject={Reject(messaging.reject, messaging.rejectUrl, onReject)}
```
to: 
```
title={consentBannerConfig.title}
reject={Reject(consentBannerConfig.reject, consentBannerConfig.rejectUrl, onReject)}
```
which feels easier to understand at a first glance. Also because `messaging.rejectUrl` could be misleading